kodilla-testing/src/main/java/com/kodilla/testing/shape/Shape.java

```diff
@@ -0,0 +1,7 @@
+package com.kodilla.testing.shape;
+
+public interface Shape {
+
+    void getShapeName();
```

Comment:
Gettery cos zwracaja chyba, co nie? Podpowiem, ze zarowno jakis wymiar jak i pole czy obwod to moga byc dowolne liczby rzeczywiste.